stdlib/public/core/SmallString.swift

```diff
@@ -108,27 +99,21 @@ extension _SmallString {
   @inlinable
   internal var isASCII: Bool {
     @inline(__always) get {
-      return discriminator.smallIsASCII
+      return _StringObject.getSmallIsASCII(fromRaw: rawDiscriminatedObject)
     }
   }
 
   // Give raw, nul-terminated code units. This is only for limited internal
   // usage: it always clears the discriminator and count (in case it's full)
-  @inlinable
+  @inlinable @inline(__always)
   internal var zeroTerminatedRawCodeUnits: RawBitPattern {
-    @inline(__always) get {
-      return (
-        self._storage.0,
-        self._storage.1 & _StringObject.Nibbles.largeAddressMask)
-    }
+    return (self._storage.0, self._storage.1 & 0x00FF_FFFF_FFFF_FFFF)
   }
 
-  @inlinable
   internal func computeIsASCII() -> Bool {
-    // TODO(String micro-performance): Evaluate other expressions, e.g. | first
     let asciiMask: UInt64 = 0x8080_8080_8080_8080
```

Comment:
I would prefer to see this named constant used uniformly everywhere, but I'm OK with keeping it out of this PR.

Reply:
Right, if it's in more than one place I agree. Otherwise, not sure about the tradeoff of baking it into our ABI and having an extra level of indirection, unless there is a lot of semantic value in the name.

Reply:
Err, never mind on the ABI part, as this isn't inlined.

Reply:
Definitely don't think we need it to be anywhere in the ABI =)